src/components/cylc/gscan/GScan.vue

```diff
@@ -59,22 +59,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         >
           <v-card width="500px">
             <v-list>
-              <v-list-item
-                v-for="(items, title) in filters"
-                :key="title"
-              >
-                <v-select
-                  v-model="filters[title]"
-                  :items="$options.allStates[title]"
-                  :label="`Filter by ${title}`"
-                  density="default"
-                  chips
-                  closable-chips
-                  clearable
-                  multiple
-                  class="my-2"
```

Comment:
The loss of this margin is causing display issues

![image](https://github.com/cylc/cylc-ui/assets/61982285/30764f87-d804-48ad-8a73-7928de784271)